types/airbnb-prop-types/index.d.ts

```diff
@@ -1,10 +1,11 @@
-// Type definitions for airbnb-prop-types 2.11
+// Type definitions for airbnb-prop-types 2.13
 // Project: https://github.com/airbnb/prop-types
 // Definitions by: Miles Johnson <https://github.com/milesj>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
 import * as PropTypes from 'prop-types';
+import * as React from 'react';
```

Comment:
We purposefully avoided coupling the `@types/react` package (as seems by all the interfaces below). Can you add a `RefLike` instead?

Reply:
Sure